### ISSUE TITLE
Postgre DB default LATIN1 encoding leads to "Failure to deploy war"

### ISSUE THREAD

**Issue by wildriff66** [h-mdm/hmdm-server#85]
OS: Ubuntu 18.04 LTS - with Tomcat8 installed

This issues is not really an issue, but only annoying.

**Issue:**
After a fresh OS installation and everything set up correctly, it still threw "Failure to deploy error". I tried a few times the install script, still the same error. But after checking the catalina.out log, I've noticed the app requires to have a database encoded with UTF8 and not with the LATIN1 default encoding.

**Solved by:**
I've changed the encoding of "hmdm" database to UTF8 from LATIN 1 and the installation and deployment worked.

**Comment by h-mdm** [comment]
You are right, Headwind MDM requires UTF-8 database encoding which is currently a standard encoding on most PostgreSQL installations (apparently beginning with Ubuntu 20.04).
